cg/store/api/findbusinessdata.py

```diff
@@ -289,21 +285,21 @@ def link(self, family_id: str, sample_id: str) -> models.FamilySample:
             .first()
         )
 
-    def links(self, case_id: str, sample_id: str, ticket: int) -> Query:
-        """Find a link between a family and a sample."""
-
-        query = self.FamilySample.query.join(models.FamilySample.family, models.FamilySample.sample)
-
-        if case_id:
-            query = query.filter(models.Family.internal_id == case_id)
-
-        if sample_id:
-            query = query.filter(models.Sample.internal_id == sample_id)
-
-        if ticket:
-            query = query.filter(models.Sample.ticket_number == ticket)
-
-        return query
+    # def links(self, case_id: str, sample_id: str, ticket: int) -> Query:

```

Comment:
This code is not used anywhere. Should it be kept or removed in you opinion?

Reply:
Code that is not used should be removed or be put in use again. Otherwise, It will just clutter the code base and cause confusion